admin/routes/helpdesk.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const express = require('express')
+const router = express.Router()
+const isAuthenticated = require('../authentication/middleware')
+const rolesConfig = require('../roles-config')
+const helpdeskController = require('../controllers/helpdesk')
+
+router.get('/', isAuthenticated(rolesConfig.ROLE_HELPDESK), (req, res, next) => helpdeskController.getSchoolImpersonation(req, res, next))
```

Comment:
Shouldn't we be using the new roles.js object that defines all the roles in the system?  Prefix with ROLE seems superfluous to me.

Reply:
@jon-shipley I could tackle that on the follow up PBI. seems like this piece of work is already doing too much.